web/contact/testdata/modify.json

```diff
@@ -2136,45 +2136,13 @@
                             "language": "eng"
                         }
                     ]
-                },
```

Comment:
this test wasn't doing what it claimed because valkey was being reset and the lock lost